LICENSE.md

```diff
@@ -70,9 +71,9 @@ their own licenses:
 
 The following components of Julia's standard library have separate licenses:
 
-- base/fftw.jl (see [FFTW](http://fftw.org/doc/License-and-Copyright.html))
```

Comment:
I just want positive confirmation from Jeff on this, rather than implicit silence, since he originally wrote this line.